packages/ilios-common/addon/routes/course/index.js

```diff
@@ -19,6 +21,9 @@ export default class CourseIndexRoute extends Route {
 
   beforeModel(transition) {
     this.session.requireAuthentication(transition, 'login');
+    if (!this.currentUser.performsNonLearnerFunction) {
```

Comment:
I'm pretty sure the `beforeModel` hook in the course route will fire and redirect before this one is hit making this code unnecessary. The same is true of the `this.session.requireAuthentication(transition, 'login');` so maybe better to have it in both places and be sure?